File: programmers/Level. 1/내적.py
# 프로그래머스 Lv.1 월간 코드 챌린지 시즌1 : 내적

def solution(a, b):
    answer = 0 # 내적
    
    # 반복문으로 타고들어가 내적을 구함
    for i in range(len(a)) :
        answer += a[i] * b[i]
    
    return answer

File: programmers/Level. 1/내적.cpp
// 프로그래머스 Lv.1 월간 코드 챌린지 시즌1 : 내적

#include <string>
#include <vector>

using namespace std;

int solution(vector<int> a, vector<int> b) {
    int answer = 0;
    
    for(int i = 0; i < a.size(); i++) {
        answer += a[i] * b[i];
    }
    
    return answer;
}